AllReadyApp/Web-App/AllReady/Security/ClaimsExtensions.cs

```diff
@@ -1,13 +1,20 @@
 ﻿using System;
 using System.Linq;
 using System.Security.Claims;
+using System.Threading.Tasks;
 using AllReady.Models;
 using Microsoft.AspNetCore.Identity;
 
 namespace AllReady.Security
 {
     public static class ClaimsExtensions
     {
+        public static async Task<string> GetUserId(this ClaimsPrincipal claimsPrincipal, UserManager<ApplicationUser> userManager)

```

Comment:
As per earlier comment, not sure if we need this. Instead use the UserManager.HetUserId method?